gdpr/models.py

```diff
@@ -142,11 +168,43 @@ class LegalReason(SmartModel):
     )
 
     @property
-    def purpose(self):
-        return purposes_map.get(self.purpose_slug, None)
+    def purpose(self) -> Type["AbstractPurpose"]:
+        return purpose_register.get(self.purpose_slug, None)
+
+    def _anonymize_obj(self, *args, **kwargs):
+        purpose_register[self.purpose_slug]().anonymize_obj(self.source_object, self, *args, **kwargs)
+
+    def _deanonymize_obj(self, *args, **kwargs):
+        purpose_register[self.purpose_slug]().deanonymize_obj(self.source_object, *args, **kwargs)
+
+    def _expirement(self):
```

Comment:
co to má vyjadřovat, ten název nezní moc anglicky?

Reply:
Má to být v češtině expirace jako průběhový čas od expire ale nevím jestli to existuje

Reply:
@Formulka 
Ale na google to má 200 k výsledků jen teda v žádném slovníku to není jen v Urban Dictionary a tam to má poněkud jiný význam https://www.urbandictionary.com/define.php?term=expirement@for